charts/pmm/values.yaml

```diff
@@ -211,7 +215,7 @@ extraLabels: {}
 ## @param serviceAccount.name Name of the service account to use. If not set and create is true, a name is generated using the fullname template.
 ##
 serviceAccount:
-  create: true
+  create: false
```

Comment:
Not sure if this change is intentional, everything else looks good

Reply:
yes, that's intentional